model/ids.go

```diff
@@ -252,3 +255,21 @@ func (s *SpanID) UnmarshalJSON(data []byte) error {
 func (s *SpanID) UnmarshalJSONPB(_ *jsonpb.Unmarshaler, b []byte) error {
 	return s.UnmarshalJSON(b)
 }
+
+// ------- OrgId -------
+
+func NewOrgId(orgId uuid.UUID) OrgId {
+	return OrgId(orgId)
+}
+
+func (o OrgId) String() string {
+	return uuid.UUID(o).String()
+}
```

Comment:
what is dis?

Reply:
a `toString()` method for the `OrgId` struct 🤦🏼‍♂️